migrate.py

```diff
@@ -208,10 +286,43 @@ def fail(msg: str) -> NoReturn:
             else:
                 new_paths = art_paths
 
+            if windowsToLinuxPath:
+                new_paths = new_paths.replace("\\", "/")
+            if linuxToWindowsPath:
+                new_paths = new_paths.replace(
+                    "/",
+                    "\\",
+                )
+
             cursor.execute(
                 "UPDATE album SET embed_art_path = ?, paths = ? WHERE id = ?",
                 (new_embed_path, new_paths, id),
             )
+        # Update albums image_files column
+        albums = cursor.execute("SELECT id, image_files FROM album").fetchall()
```

Comment:
Rather than override the same variable and use a fake third column, it would probably be better to make a new variable here `album_images: List[Tuple[str, str]]` or something...`